src/test/java/project/eepw/softwaretestingcrud/domain/DomainUtils.java

```diff
@@ -0,0 +1,31 @@
+package project.eepw.softwaretestingcrud.domain;
+
+import project.eepw.softwaretestingcrud.domain.post.entity.Post;
+import project.eepw.softwaretestingcrud.domain.user.entity.User;
+
+import java.util.Collections;
+
+public final class DomainUtils {
```

Comment:
Also this method name is quite strange beacuse it is so wide and I do not know actually what this class is created for. If we really want to have such class, maybe think about some better naming than this